deployment/chainloop/templates/controlplane/secret-config.yaml

```diff
@@ -29,24 +29,26 @@ stringData:
   {{- if and .Values.controlplane.keylessSigning.enabled (eq "fileCA" .Values.controlplane.keylessSigning.backend) }}
   fileca.secret.yaml: |
     {{- with .Values.controlplane.keylessSigning.fileCA }}
-    certificate_authority:
-      file_ca:
-        cert_path: "/ca_secrets/file_ca.cert"
-        key_path:  "/ca_secrets/file_ca.key"
-        key_pass: "{{- required "FileCA keyPass is mandatory" .keyPass }}"
+    certificate_authorities:
+      - issuer: true
+        file_ca:
+          cert_path: "/ca_secrets/file_ca.cert"
+          key_path:  "/ca_secrets/file_ca.key"
+          key_pass: "{{- required "FileCA keyPass is mandatory" .keyPass }}"
     {{- end }}
   {{- end }}
   {{- if and .Values.controlplane.keylessSigning.enabled (eq "ejbcaCA" .Values.controlplane.keylessSigning.backend) }}
   ejbca.secret.yaml: |
     {{- with .Values.controlplane.keylessSigning.ejbcaCA }}
-    certificate_authority:
-      ejbca_ca:
-        cert_path: "/ca_secrets/ejbca_client.cert"
-        key_path:  "/ca_secrets/ejbca_client.key"
-        server_url: "{{- required "EJBCA server URL is mandatory" .serverURL }}"
-        certificate_profile_name: "{{- required "EJBCA certificate profile name is mandatory" .certProfileName }}"
-        end_entity_profile_name: "{{- required "EJBCA end entity profile name is mandatory" .endEntityProfileName }}"
-        certificate_authority_name: "{{- required "EJBCA certificate authority name is mandatory" .caName }}"
+    certificate_authorities:
```

Comment:
should we add the issuer option in the values.yaml file too? Or should we do it later on to support multiple CASs?

Reply:
Yes, I didn't want to disrupt current values.yaml. I would add it when we support multiple CAs.